SignalServiceKit/src/Contacts/TSThread.h

```diff
@@ -157,18 +157,19 @@ NS_ASSUME_NONNULL_BEGIN
 @property (atomic, readonly, nullable) NSDate *mutedUntilDate;
 
 // This model may be updated from many threads. We don't want to save
-// our local copy (this instance) since it may be out of date.  Instead, we
+// our local copy (this instance) since it may be out of date.  We also
+// want to avoid re-saving a model that has been deleted.  Therefore, we
 // use these "updateWith..." methods to:
 //
 // a) Update a property of this instance.
-// b) Load an up-to-date instance of this model from from the data store.
-// c) Update and save that fresh instance.
-// d) If this instance hasn't yet been saved, save this local instance.
+// b) If a copy of this model exists in the database, load an up-to-date copy,
+//    and update and save that copy.
+// b) If a copy of this model _DOES NOT_ exist in the database, do _NOT_ save
+//    this local instance.
 //
 // After "updateWith...":
 //
-// a) An updated copy of this instance will always have been saved in the
-//    data store.
+// a) An copy of this model in the database will have been updated.
```

Comment:
typo: "An copy" -> "Any copy"? 

Apart from the grammar error, I idiomatically use "Any" to connote that a thing might not exist.

Reply:
Done.